lib/en/managing-domains-ssl.adoc

```diff
@@ -120,10 +120,10 @@ link:#top[Back to Top]
 === Using a Custom SSL Certificate
 OpenShift includes support for link:http://en.wikipedia.org/wiki/Server_Name_Indication[Server Name Identification], which improves support for link:http://en.wikipedia.org/wiki/Server_Name_Indication#How_SNI_fixes_the_problem[TLS] by sending your OpenShift-configured domain alias as a part of the handshake.
 
-You can always take advantage of our **.rhcloud.com* wildcard certificate in order to securely connect to any application via it's original, OpenShift-provided hostname URL.
-
 Support for enabling *HTTPS* connections to custom, aliased hostnames is available for users of https://www.openshift.com/products/pricing[OpenShift Online's premium plans].
 
+If you are not using one of the premium plans, or when connecting to your app using link:managing-port-binding-routing.html[secure web sockets], you can always take advantage of our **.rhcloud.com* wildcard certificate in order to securely connect to any application via it's original, OpenShift-provided hostname URL.
```

Comment:
What about: 

If you are not using one of the premium plans, or if you are connection to your app using secure web sockets, you can take advantage of ` (...)` ?